### DISCUSSION TITLE
Add screenshots for each example

### DISCUSSION THREAD

**Discussion by MarkOSullivan94** [simolus3/zap#14] [Ideas]
I stumbled upon the zap package this morning and was curious about it but frustrated I couldn't find more examples of websites built using zap.

It'd be great if you could add examples in each of the example directories to what the website looks like

https://github.com/simolus3/zap/tree/main/examples

Also side question - is the zap website build using zap? 

https://simonbinder.eu/zap/

**Comment by simolus3** [comment]
> It'd be great if you could add examples in each of the example directories to what the website looks like

Yeah the top-level `examples/` folder is rather outdated, I've removed it. The examples also shown on the website are better resources.

> Also side question - is the zap website build using zap?

Most of it is static content. The example view at https://simonbinder.eu/zap/examples/ is is a zap component though, it's sources are [here](https://github.com/simolus3/zap/tree/main/docs/lib/src/examples).
